events-api/src/modules/tickets/tickets.controller.ts

```diff
@@ -0,0 +1,88 @@
+import { TicketsService } from './tickets.service';
+import { AuthTokenGuard } from 'src/shared/guards/auth-token.guard';
+import { Body, Controller, Post, UseGuards, Request, Get, Param, ParseUUIDPipe, HttpStatus, Delete, Req, Put } from '@nestjs/common';
+import { CreateTicketDto } from './dto/requests/create-ticket.dto';
+import { PayTicketDto } from './dto/requests/pay-ticket.dto';
+
+@Controller('tickets')
+export class TicketsController {
+  constructor(private readonly ticketService: TicketsService) {}
+
+  @Post()
+  @UseGuards(AuthTokenGuard)
+  async createTicket(@Body() createTicketDto: CreateTicketDto, @Request() req) {
+    const userId = req.user.userId;
+    const createTicket = await this.ticketService.createTickets(createTicketDto, userId);
+
+    return {
+      data: createTicket,
+    };
+  }
+
+  @Get()
+  @UseGuards(AuthTokenGuard)
+  async getUserTickets(@Request() req) {
```

Comment:
Don't use req directly. If you want to access user of the request, instead use decorators